src/org/loklak/api/search/QuoraProfileScraper.java

```diff
@@ -87,9 +102,40 @@ public static SusiThought  scrapeQuora(String profile) {
 		String rssFeedLink = url + "/rss";
 		quoraProfile.put("rss_feed_link", rssFeedLink);
 
+		Elements userBasicInfo = userHTML.getElementsByAttributeValueContaining("class", "UserCredential IdentityCredential");
+		for (Element info: userBasicInfo) {
+			String infoText = info.text();
+			if (infoText.startsWith("Works")) {
+				quoraProfile.put("works_at", infoText);
```

Comment:
works_at works fine for this case (the profile present in the screenshot). But i think in few cases it may not work. For eg : https://www.quora.com/profile/Samriddhi-Sinha . In this profile it is written as Student developer at. Will the scraper work here?